source/developers/plugins/plugin-descriptor.rst

```diff
@@ -110,32 +110,35 @@ to your ``craftercms-plugin.yaml`` descriptor file
    installation:
     - type: preview-app
       parentXpath: //widget[@id='craftercms.components.ToolsPanel']
-      testXpath: //plugin[@id='org.craftercms.plugin.sidebar']
+      elementXpath: //plugin[@id='org.craftercms.sampleComponentLibraryPlugin.components.reactComponent']
       element:
         name: configuration
         children:
         - name: widgets
           children:
           - name: widget
+            attributes:
+            - name: id
+              value: org.craftercms.sampleComponentLibraryPlugin.components.reactComponent
             children:
             - name: plugin
               attributes:
               - name: id
-                value: org.craftercms.plugin.sidebar
+                value: org.craftercms.plugin
```

Comment:
Is this the right value? this should match the plugin id used at the top of the yaml